### PR TITLE
Superheroes update

### PR DESCRIPTION
**Pull request by ZomgPonies**
:cl:
fix: Fixes Superhero ID cards
fix: Fixes Griffin missing his Freedom Implant
fix: Fixes ElectroNegmatic's Ability not working at all.
tweak: Changes LightnIan's ability into a non-damaging stun lasting 2 ticks, with the amount of people affected depending on how long you hold the power in for.
/:cl:

- New PR due to lead merges
- Cleans up useless items only used for superheros

### CODE REVIEW

**Comment by ZomgPonies** [comment]
@Fox-McCloud I switched from a 2 tick stun to a 3 tick slowed, as I really don't want to disarm people with this hero. (That's what E-N's power is about).

File: code/datums/spells/lightning.dm [hunk 1]
```diff
@@ -15,10 +15,12 @@
 	var/image/halo = null
 	action_icon_state = "lightning"
 	var/sound/Snd // so far only way i can think of to stop a sound, thank MSO for the idea.
+	var/damaging=TRUE
```
Comment (tigercat2000) [inline code/datums/spells/lightning.dm]: operator spacing please

File: code/datums/spells/magnet.dm [hunk 2]
```diff
@@ -12,34 +12,37 @@
...
 	var/ready = 0
+	var/start_time = 0
 	var/image/halo = null
+	var/sound/Snd // so far only way i can think of to stop a sound, thank MSO for the idea.
 	action_icon_state = "tech"
 
 
 /obj/effect/proc_holder/spell/targeted/magnet/Click()
-	if(!ready)
+	if(!ready && start_time==0)
```
Comment (tigercat2000) [inline code/datums/spells/magnet.dm]: operator spacing please

File: code/datums/spells/magnet.dm [hunk 3]
```diff
@@ -12,34 +12,37 @@
...
 	else
-		if(cast_check(skipcharge=1))
+		if(ready && cast_check(skipcharge=1))
 			choose_targets()
 	return 1
 
 /obj/effect/proc_holder/spell/targeted/magnet/proc/StartChargeup(mob/user = usr)
 	ready = 1
 	to_chat(user, "<span class='notice'>You start gathering the power.</span>")
+	Snd = new/sound('sound/magic/lightning_chargeup.ogg',channel = 7)
```
Comment (tigercat2000) [inline code/datums/spells/magnet.dm]: spaces after commas in arguments please

File: code/datums/spells/magnet.dm [hunk 4]
```diff
@@ -12,34 +12,37 @@
...
 	user.overlays.Add(halo)
-	spawn(0)
-		while(ready)
-			sleep(1)
-			energy++
-			if(energy >= 100 && ready)
-				Discharge()
+	playsound(get_turf(user), Snd, 50, 0)
+	start_time = world.time
+	if(do_mob(user,user,100,uninterruptible=1))
```
Comment (tigercat2000) [inline code/datums/spells/magnet.dm]: comma spacing please

File: code/datums/spells/magnet.dm [hunk 5]
```diff
@@ -50,16 +53,16 @@ obj/effect/proc_holder/spell/targeted/magnet/proc/Reset(mob/user = usr)
...
 
 
 /obj/effect/proc_holder/spell/targeted/magnet/cast(list/targets, mob/user = usr)
-
-	var/mob/living/carbon/target = targets[1]
-
+	ready = 0
+	var/mob/living/target = targets[1]
+	Snd=sound(null, repeat = 0, wait = 1, channel = Snd.channel) //byond, why you suck?
+	playsound(get_turf(user),Snd,50,0)// Sorry MrPerson, but the other ways just didn't do it the way i needed to work, this is the only way.
```
Comment (tigercat2000) [inline code/datums/spells/magnet.dm]: more spacing

File: code/datums/spells/magnet.dm [hunk 6]
```diff
@@ -50,16 +53,16 @@ obj/effect/proc_holder/spell/targeted/magnet/proc/Reset(mob/user = usr)
...
 	start_recharge()
 
 
 /obj/effect/proc_holder/spell/targeted/magnet/cast(list/targets, mob/user = usr)
-
-	var/mob/living/carbon/target = targets[1]
-
+	ready = 0
+	var/mob/living/target = targets[1]
+	Snd=sound(null, repeat = 0, wait = 1, channel = Snd.channel) //byond, why you suck?
```
Comment (tigercat2000) [inline code/datums/spells/magnet.dm]: almost missed this, the assignment operator spacing

**Comment by ktoma36** [comment]
Want to make a comment on henchmen. The toolbox that gets stuck to their arm cannot be removed by nothing other than surgery, even and admin couldn't get it off. The main reason I bring this up is that henchmen can use a wish granter like everyone else and be stuck with a toolbox permanently attached to them even though they turn into a hero/villian.

**Comment by ZomgPonies** [comment]
@ktoma36 I'll be making a separate PR to address the wishgranter once the actual archetypes are balanced and documented.

**Pull request merged**